doc/source/structures/vessels/crafttemplate.rst

```diff
@@ -0,0 +1,105 @@
+.. _crafttemplate:
+
+CraftTemplate
+=============
+
+.. structure:: CraftTemplate
+
+    You can access :struct:`CraftTemplate` objects from the :struct:`KUniverse`
+    bound variable.  Templates can be used to launch new vessels, or read initial
+    data about a craft, such as the description.
+
+    .. list-table::
+        :header-rows: 1
+        :widths: 1 1 2
+
+        * - Suffix
+          - Type
+          - Description
+
+        * - :attr:`NAME`
+          - :struct:`String`
+          - Name of this craft template
+        * - :attr:`FILEPATH`
+          - :struct:`String`
+          - The path to the craft file
+        * - :attr:`DESCRIPTION`
+          - :struct:`String`
+          - The description as saved in the editor
+        * - :attr:`EDITOR`
+          - :struct:`String`
+          - The editor where this craft was saved
+        * - :attr:`LAUNCHSITE`
+          - :struct:`String`
+          - The default launch site for this craft
+        * - :attr:`MASS`
+          - :struct:`Scalar`
+          - The default mass of the craft
+        * - :attr:`COST`
+          - :struct:`Scalar`
+          - The default cost of the craft
+        * - :attr:`PARTCOUNT`
+          - :struct:`Scalar`
+          - The total number of parts n this craft.
+
+
+.. attribute:: CraftTemplate:NAME
+
+    :access: Get only
+    :type: :struct:`String`
+
+    Returns the name of the craft.  It may differ from the file name.
+
+.. attribute:: CraftTemplate:FILEPATH
+
+    :access: Get only
+    :type: :struct:`String`
+
+    Returns the absolute file path to the craft file.
+
+.. attribute:: CraftTemplate:DESCRIPTION
+
+    :access: Get only
+    :type: :struct:`String`
+
+    Returns the description field of the craft, which may be edited from the
+    drop down window below the craft name in the editor.
+
+.. attribute:: CraftTemplate:EDITOR
+
+    :access: Get only
+    :type: :struct:`String`
+
+    Name of the editor from which the craft file was saved.  Valid values are
```

Comment:
I find the notion of it being a relic of typewriters odd.  Because I find it even MORE important when using variable-width fonts than monospaced ones.  (Thus why I'm annoyed that HTML rendering prevents it.)  Variable-width fonts have narrower period characters than monospaced ones, making the end of the sentence stand out even less than it does in old-school typewriters.